java/code/src/com/redhat/rhn/manager/errata/ErrataManager.java

```diff
@@ -637,11 +641,16 @@ private static void deleteErrata(User user, List<OwnedErrata> erratas) {
      */
     public static void deleteErratum(User user, Errata errata) {
         List<OwnedErrata> eids = new ArrayList<>();
+        OwnedErrata oErrata = buildOwnedErrataFromErrata(errata);
+        eids.add(oErrata);
+        deleteErrata(user, eids);
+    }
+
+    private static OwnedErrata buildOwnedErrataFromErrata(Errata errata) {
         OwnedErrata oErrata = new OwnedErrata();
         oErrata.setId(errata.getId());
         oErrata.setAdvisory(errata.getAdvisory());
```

Comment:
Nitpicking here, but I would move this code into a `OwnedErrata` construct, so that we can just call `new OwnedErrata(errata)`.